versioned_docs/version-7.x/stack-navigator.md

```diff
@@ -231,6 +231,21 @@ Use this prop to have a semi-transparent dark overlay visible under the card dur
 
 Function which returns a React Element to display as the overlay for the card. Make sure to set `cardOverlayEnabled` to `true` when using this.
 
+<video playsInline autoPlay muted loop>
+  <source src="/assets/7.x/stack/cardOverlay.mp4" />
+</video>
+
+Example:
+
+```js
+    screenOptions={{
+      cardOverlayEnabled: true,
+      cardOverlay: () => (
+        <View style={{ flex: 1, backgroundColor: 'gray', opacity: 0.5 }} />
+      ),
+    }}
```

Comment:
Please pay attention, whether this should be called `screenOptions` or `options`. For single screen, prop that refers to options is called just `options`, whether for navigator it is called `screenOptions`. Can we also extend this example a bit, so that it will screen definition?

```suggestion
<Stack.Screen
  name="Article"
  component={Article}
  options={{
    cardOverlayEnabled: true,
    cardOverlay: () => (
      <View style={{ flex: 1, backgroundColor: 'rgba(0, 0, 0, 0.15)' }} />
    ),
  }}
/>
```

Something like this, you can get inspired by `Animation related options` subsection 😄

Reply:
@satya164 let me know what do you think about this suggestion with example. Do you think we should keep definitions of screen on every code snippet, or we can leave it? What do you think looks better?